### PR TITLE
Fixing margin on top of nav-bar

### PR DESCRIPTION
**Pull request by akankshadharkar**
main.css has a rule for nav:
```
nav {
  width: 100%;
  text-align: right;
  margin-top: 10px;
}
```
which is the reason for the white-line (margin) on top of the navbar

### CODE REVIEW

**Review by swarajpure — APPROVED** [review]

**Comment by akankshadharkar** [comment]
Screenshot 2020-11-10 at 22 02 12

File: js/navbar.js [hunk 1]
```diff
@@ -0,0 +1,8 @@
+function changeCSS() {
+  const elementToManipulate = document.getElementById("topNavBar");
+  if (elementToManipulate.className === "topnav") {
+    elementToManipulate.className += " responsive";
```
Comment (ankushdharkar) [inline js/navbar.js:4]: Nit: Space should be it's own character, so it's clear that it was put on purpose
```js
elementToManipulate.className += " " + "responsive";
```

(Thank god we have string interpolation in modern ways)

File: index.html [hunk 2]
```diff
@@ -106,6 +109,7 @@ <h3 class="info-note"> To know how this page is developed,
     </footer>
   </div>
   <script src="js/main.js"></script>
+  <script src="js/navbar.js"></script>
 </body>
 
 </html>
```
Comment (ankushdharkar) [inline index.html:115]: Newline missing at end

image

File: js/navbar.js [hunk 3]
```diff
@@ -0,0 +1,8 @@
...
+  } else {
+    elementToManipulate.className = "topnav";
+  }
+}
```
Comment (ankushdharkar) [inline js/navbar.js:8]: New line missing

File: css/navbar.css [hunk 4]
```diff
@@ -1,27 +1,60 @@
...
+    position: absolute;
+    right: 0;
+    top: 0;
+  }
+  .topnav.responsive a {
+    float: none;
+    display: block;
+    text-align: left;
+  }
 }
```
Comment (ankushdharkar) [inline css/navbar.css:60]: New line missing 
image

**Review by ankushdharkar — CHANGES_REQUESTED** [review]

File: css/navbar.css (hunk 4 above)
Comment (akankshadharkar) [reply css/navbar.css:60]: Just curious why we want to surely leave a new line after the main closing in our template files

**Comment by akankshadharkar** [comment]
I think this PR is valid as currently we show the navbar like this:
Screen Shot 2021-04-01 at 2 39 57 PM

Making it a hamburger icon will enhance UX, IMO 
But I am in the process of trying to get new designs for a consistent navbar throughout all our sites for better discoverability
Maybe we should keep this on hold till then?

**Comment by rohan09-raj** [comment]
There are no further developments in the PR. That's why closing it.

**Pull request closed without merging**